src/main/webapp/scripts/map.js

```diff
@@ -80,12 +88,79 @@ window.initMap = function () {
         strokeOpacity: 0.8,
         strokeWeight: 2
       });
+      // Once cards are retrieved...
+      // Add vendor's markers
+      addVendorsToMap(map);
     },
-    (error) => {
-      if (error.code == error.PERMISSION_DENIED)
-        alert('Permission denied to access location');
-    });
+      (error) => {
+        if (error.code == error.PERMISSION_DENIED)
+          alert('Permission denied to access location');
+      });
   } else {
     alert('This browser does not support location');
   }
+};
+
+/**
+ * Function that adds vendor marks to the map
+ */
+const addVendorsToMap = (map) => {
```

Comment:
Also here is it possible to add the JSDoc for map?

Reply:
The same here, how would you describe it on a JSDoc format ?

Reply:
I think maybe as an object, I am not sure. Haha

Reply:
Yup, I'd probably just do * @param {Object} map